Formula/sysstat.rb

```diff
@@ -1,8 +1,8 @@
 class Sysstat < Formula
   desc "Performance monitoring tools for Linux"
   homepage "https://github.com/sysstat/sysstat"
-  url "https://github.com/sysstat/sysstat/archive/v12.4.0.tar.gz"
-  sha256 "1962ed04832d798f5f7e787b9b4caa8b48fe935e854eef5528586b65f3cdd993"
+  url "https://github.com/sysstat/sysstat/archive/v12.4.1.tar.gz"
```

Comment:
There is a tag for 12.5.3. Any reason why we're not using that?

Reply:
We totally should, flip into the latest tag.